Fortnox.NET.Tests/Fortnox.NET.Tests/OrderTests.cs

```diff
@@ -194,5 +193,85 @@ public void CreateOrderTest()
 
             Assert.AreEqual(1, response.OrderRows.Count);
         }
+
+        [TestMethod]
+        public void CreateInvoiceTest()
+        {
+            var request = new FortnoxApiRequest(this.connectionSettings.AccessToken, this.connectionSettings.ClientSecret);
+
+            var createOrderResponse = OrderService.CreateOrderAsync(request,
+                new Order
+                {
+                    CustomerNumber = "1",
+                    OrderRows =
+                        new List<OrderRow>
+                        {
+                            new OrderRow
+                            {
+                                ArticleNumber = "1",
+                                OrderedQuantity = 1,
+                                AccountNumber = 3001
+                            }
+                        }
+                }).GetAwaiter().GetResult();
+            var createInvoiceResponse = OrderService.CreateInvoiceAsync(request, createOrderResponse.DocumentNumber).GetAwaiter().GetResult();
+
+            Assert.AreEqual("1", createInvoiceResponse.CustomerNumber);
```

Comment:
The CreateInvoice action sets an `InvoiceReference` for the order in, a more interesting test would be to check if it actually got a new reference to an invoice.